package.json

```diff
@@ -15,7 +15,8 @@
     "docs:build": "typedoc lib",
     "test:vm": "node ./tests/tester --vm",
     "test:state": "npm run build:dist && node ./tests/tester --state --dist",
-    "test:state:allForks": "npm run test:state -- --fork=Byzantium && npm run test:state -- --fork=Constantinople && npm run test:state -- --fork=Petersburg && npm run test:state -- --fork=Istanbul",
+    "test:state:allForks": "npm run test:state -- --fork=Byzantium && npm run test:state -- --fork=Constantinople && npm run test:state -- --fork=Petersburg && npm run test:state -- --fork=Istanbul && npm run test:state -- --fork=MuirGlacier",
+    "test:state:selectedForks": "npm run test:state -- --fork=Constantinople && npm run test:state -- --fork=Istanbul && npm run test:state -- --fork=MuirGlacier",
```

Comment:
Curious why you chose constantinope instead of petersburg here?

Reply:
Because I mixed up the fork names. 🤪 Will update in 1-2 hours.

Reply:
Or alternatively, since this is time critical: can you post the update here?

Reply:
done.